packages/lit-dev-content/site/docs/templates/expressions.md

```diff
@@ -137,14 +137,25 @@ html`<main>${bodyText}</main>`
 
 Expressions in the child position can take many kinds of values:
 
-* Primitive values likes strings, numbers, and booleans
+* Primitive values
+* Non-rendering values
 * `TemplateResult` objects created with the `html` function.
 * DOM nodes
 * Arrays or iterables of any of the supported types
 
 ### Primitive values
 
-Primitives values like strings, numbers, booleans, null, and undefined are converted to strings when interpolated into text content or attribute values. They are checked for equality with the previous value so the DOM is not updated if the value hasn't changed.
+Strings, numbers, and booleans are all converted to strings when interpolated into text content.
```

Comment:
I think this might be clear if it mentions all primitives. So add bigint, mention that symbol throws, and mention right away that `null`, `undefined`, and `''` render nothing, with a link to the next section. Possibly link to https://developer.mozilla.org/en-US/docs/Glossary/Primitive

Reply:
Incorporated.